package.json

```diff
@@ -15,7 +15,11 @@
     "commitlint": "commitlint --edit"
   },
   "dependencies": {
+    "@emotion/react": "^11.13.3",
+    "@emotion/styled": "^11.13.0",
```

Comment:
Do we need Emotion; MUI seems to have styled components built-in. See https://mui.com/system/styled/.